modules/wallet/money.go

```diff
@@ -112,6 +119,9 @@ func (w *Wallet) SendSiacoins(amount types.Currency, dest types.UnlockHash) ([]t
 		w.log.Println("Attempt to send coins has failed - failed to sign transaction:", err)
 		return nil, build.ExtendErr("unable to sign transaction", err)
 	}
+	if w.deps.disrupt("SendSiacoinsInterrupted") {
+		return nil, errors.New("Failed to accept transaction set. (SendSiacoinsInterrupted)")
```

Comment:
errors are not supposed to have punctuation, e.g. `failed to accept transaction set (SendSiacoins interrupted)`